src/cls/IPM/Repo/Remote/PackageService.cls

```diff
@@ -266,7 +266,7 @@ Method IsAvailable() As %Boolean
   Set tAvailable = 0
   Set tRequest = ..GetHttpRequest()
   
-  Set tRequest.Timeout = 30
+  Set tRequest.Timeout = 3
```

Comment:
When setting any hardcoded strings/numbers please add a comment indicating why the value is chosen